UnoCheck/InteractiveSelector.cs

```diff
@@ -0,0 +1,183 @@
+	{
+		/// <summary>
+		/// Determines if interactive prompts should be shown based on settings and environment
+		/// </summary>
+		public static bool ShouldShowInteractivePrompts(CheckSettings settings)
+		{
+			// Don't show in CI or non-interactive mode
+			if (settings.CI || settings.NonInteractive)
+				return false;
+
+			// Don't show if any relevant flags are already specified
+			if (!string.IsNullOrEmpty(settings.Ide))
+				return false;
+			
+			if (settings.TargetPlatforms?.Any() == true)
+				return false;
+			
+			if (settings.Frameworks?.Any() == true)
+				return false;
+			
+			if (settings.Skip?.Any() == true)
+				return false;
+
+			return true;
+		}
+
+		/// <summary>
+		/// Prompts user to select their IDE(s)
+		/// </summary>
+		/// <returns>
+		/// Array of IDE identifiers. If no IDEs selected, returns empty array and no IDE checks will be skipped.
+		/// </returns>
+		public static string[] PromptForIde()
+		{
+			var ideChoices = new List<string>();
+
+			// Show platform-appropriate IDEs
+			if (Util.IsWindows)
+			{
+				ideChoices.Add("Visual Studio");
+			}
+			
+			// Common IDEs across all platforms
+			ideChoices.Add("VS Code");
+			ideChoices.Add("Rider");
+			ideChoices.Add("Other / None");
+
+			var selectedIdes = AnsiConsole.Prompt(
+				new MultiSelectionPrompt<string>()
+					.Title("[bold blue]Which IDE(s) do you plan to use?[/]")
+					.PageSize(10)
+					.AddChoices(ideChoices)
+					.HighlightStyle(new Style(Color.Green)));
+
+			// If no IDEs selected, return empty array (no IDE-specific checks will be skipped)
+			if (!selectedIdes.Any())
+				return Array.Empty<string>();
+
+			// Map friendly names to internal identifiers
+			return selectedIdes.Select(ide => ide switch
+			{
+				"Visual Studio" => "vs",
+				"VS Code" => "vscode",
+				"Rider" => "rider",
+				"Other / None" => "other",
+				_ => "other"
+			}).ToArray();
+		}
+
+		/// <summary>
+		/// Prompts user to select target platforms
+		/// </summary>
+		/// <returns>
+		/// Array of platform identifiers. If empty, all platforms will be targeted.
+		/// </returns>
+		public static string[] PromptForTargetPlatforms()
+		{
+			var platformChoices = new List<string>();
+
+			// Platform order: desktop, wasm, ios, android, winappsdk
+			platformChoices.Add("Desktop");
+			platformChoices.Add("WebAssembly");
+			
+			// iOS is available on Windows and Mac only (not on Linux)
+			if (Util.IsWindows || Util.IsMac)
+			{
+				platformChoices.Add("iOS");
+			}
+			
+			platformChoices.Add("Android");
+			
+			// Windows App SDK only on Windows
+			if (Util.IsWindows)
+			{
+				platformChoices.Add("Windows App SDK");
+			}
+
+			var selectedPlatforms = AnsiConsole.Prompt(
+				new MultiSelectionPrompt<string>()
+					.Title("[bold blue]Which platforms do you want to target?[/]")
+					.PageSize(10)
+					.InstructionsText("[grey](Press [blue]<space>[/] to toggle, [green]<enter>[/] to accept. Select none to target all platforms)[/]")
```

Comment:
To avoid confusion with any `none` options/mapping with the rest of the code and making it clearer to the user

```suggestion
					.InstructionsText("[grey](Press [blue]<space>[/] to toggle, [green]<enter>[/] to accept. Select none of them to target all platforms)[/]")
```